package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "eslint lib test *.js",
     "test": "npm run lint && egg-bin test",
     "test-local": "egg-bin test",
+    "test-script": "eslint lib test *.js --fix",
```

Comment:
why add this script? you should use `npm run lint -- --fix`

Reply:
Well……

First it's the popomore's suggestion.

Second, I've referred what you've defined in the egg-security project, it seems we have a `--fix` switch defined in the package.json, but it seems it's NOT allowed to add in the lint. What do you think of this? See `"lint": "eslint --ext js . --fix"`.

Third, some problems can be fixed by this switch, I love that and it's really useful.